PilotDataPlatform/pipelines
# PILOT-2044: Update dataops client to call task-stream for job update

## Summary

- Updated dataops client to call `POST /task-stream` for updating job status.
- Removed @atexit.register decorator, which was causing an error to be raised at the end of the copy/delete pipeline that a parameter was missing.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-2044

## Type of Change

Please delete options that are not relevant.


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

- Pipeline for copy/delete process should not fail during stage of updating job id using dataops client.
